### PR TITLE
plot_function for the 'model_quantities' (issue #673)

### PR DESCRIPTION
**Pull request by suyashgupta20**
Similar to the spectrum I have created a function `plot_model_quantities` which returns the plot corresponding to every `model_quantities` to the test function, `test_model_quantities`

### CODE REVIEW

File: tardis/tests/integration_tests/test_integration.py [hunk 1]
```diff
@@ -154,6 +156,34 @@ def test_model_quantities(self, model_quantities):
         tardis_quantity = eval('self.result.' + tardis_quantity_name)
         assert_allclose(tardis_quantity, reference_quantity)
 
+    def plot_model_quantities(self, model_quantities):
+        reference_quantity_name, tardis_quantity_name = model_quantities
+        reference_quantity = self.reference[reference_quantity_name]
+        tardis_quantity = eval('self.result.' + tardis_quantity_name)
+
+        gs_mq = (plt.GridSpec(2, 1, height_ratios=[3, 1]))
+        model_ax_mq = (plt.subplot(gs_mq[0]))
```
Comment (karandesai-96) [inline tardis/tests/integration_tests/test_integration.py]: You don't have to enclose these calls in parenthesis. Same thing applies to other occurrences as well.

Comment (suyashgupta20) [inline tardis/tests/integration_tests/test_integration.py]: okay, I will change this in my next commit.

**Comment by suyashgupta20** [comment]
image
I got these type of plots in my report.

**Comment by suyashgupta20** [comment]
@yeganer  @unoebauer...what are your views on this.

**Comment by karandesai-96** [comment]
As the issue states, there are a certain set of scalars in model quantities which do not make sense to plot, so I think you should first add legends and a pass/ fail badge to the plot, and the next thing to do is to create a fixture parametrized y scalars only.

So you see, there would be `test_model_quantities`, `test_model_scalars`, `plot_model_quantities` but not `plot_model_scalars`.

**Comment by suyashgupta20** [comment]
Okay @karandesai-96  i will make that changes.
actually i got the error for the values like `/simulation/plasma/level_number_density` in every reference fie that is at.h5, paper1.scatter.h5 etc. What we have to do for the values like this. @unoebauer

**Comment by suyashgupta20** [comment]
@karandesai-96 I added fixture for model_scalars but the problem is that it's actual value (tardis_quantity) is not define inside the montecarlo and plasma like other values. so what we have to do for that ?

**Pull request closed without merging**